### DISCUSSION TITLE
New NPM package name: `@celo/compliance` (formerly, `compliance-sdk`)

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/compliance#19] [Announcements]
## TLDR

- The [`compliance-sdk`](https://www.npmjs.com/package/compliance-sdk) package has moved to [`@celo/compliance`](https://www.npmjs.com/package/@celo/compliance)
- This is only a **package name** change. The goal is to include it in the `@celo/*` scope for security purposes and to avoid confusion.
- The functionality in [`@celo/compliance`](https://www.npmjs.com/package/@celo/compliance) is exactly the same as it was in [`compliance-sdk`](https://www.npmjs.com/package/compliance-sdk)

## What does this mean?

1. If you are currently using [`compliance-sdk`](https://www.npmjs.com/package/compliance-sdk) as a dependency in your code base, please replace it with [`@celo/compliance`](https://www.npmjs.com/package/@celo/compliance).
Everything else can remain unchanged.

2. The [`compliance-sdk`](https://www.npmjs.com/package/compliance-sdk) package shows a warning message to encourage users to switch [`@celo/compliance`](https://www.npmjs.com/package/@celo/compliance).
    <img width="1190" alt="image" src="https://github.com/celo-org/compliance/assets/46296830/f221f8dc-b434-47f3-8573-24779a92f883">

3. If you have any questions, feel free to comment below. If you have a particularly sensitive issue please reach out to [devrel@celo.org](mailto:devrel@celo.org).
